Sources/carton/Toolchain/ToolchainManagement.swift

```diff
@@ -282,4 +282,18 @@ extension FileSystem {
 
     return AbsolutePath(binPath)
   }
+  
+  func fetchAllSwiftVersion() throws -> [String] {
+    var versions: [String] = []
+    // sdks installed by carton
+    let cartonSdkPath = homeDirectory.appending(components: ".carton", "sdk")
```

Comment:
**suggestion:** could you separate out `homeDirectory.appending(components: ".carton", "sdk")` into a computed property within this same extension? Computing this path already happens in this same file on line 126, would be great to avoid duplication. Also, a more suitable name would be `cartonSDKPath` to keep the capitalization consistent.